dcos/dcos-ui
# DCOS-43911 SDK Plans v1

## Testing

1. Install a SDK package e.g. `cassandra`|`confluent-kafka` etc.
2. Visit the `services/detail` page for the installed service
3. Ensure the `Plans` tabs exists
4. Visit the `Plans` tab
5. Ensure there is a dropdown of plans
6. Plan details should be displayed in table below the dropdown
7. Change plan selected in dropdown
8. Plan details in table should update.

## Trade-offs

N/A

## Dependencies

None

## Screenshots

![image](https://user-images.githubusercontent.com/1972968/51351461-8ca82580-1a70-11e9-80f1-bdbf64b5f087.png)
![image](https://user-images.githubusercontent.com/1972968/51351495-9e89c880-1a70-11e9-8615-45adee239869.png)
![image](https://user-images.githubusercontent.com/1972968/51351540-bbbe9700-1a70-11e9-9ac8-7af4c9afa3d3.png)
